integr8ly/integreatly-operator
# Removed rhoam- prefix from AlertManager, Grafana, and Prometheus CR names

# Issue link
https://issues.redhat.com/browse/MGDAPI-2898

# What
Removed `rhoam-` prefix from observability AlertManager, Grafana, and Prometheus CR names.


# Verification steps

- If all test pass that should be sufficient verification but alternatively checkout this PR and install RHOAM onto cluster. Verify the installation successfully completes without error.
